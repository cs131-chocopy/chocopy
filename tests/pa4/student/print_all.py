a:object=None
b:int=114514
c:bool=True
d:str="haha"
a=b
print(a)
a=c
print(a)
a=d
print(a)

