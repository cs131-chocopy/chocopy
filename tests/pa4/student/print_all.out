114514
True
haha
